cfgov/legacy/templates/hud/housing_counselor_pdf_selfcontained.html

```diff
@@ -0,0 +1,281 @@
+{% load staticfiles %}
```

Comment:
I'm no front-end expert but is it possible for the js and css to be split out of this?